data/data/bootstrap/systemd/units/release-image-pivot.service.template

```diff
@@ -3,7 +3,7 @@
 Description=Pivot bootstrap to the OpenShift Release Image
 Wants=release-image.service
 After=release-image.service
-{{- if .BootstrapInPlace }}
+{{- if or (.BootstrapInPlace) (eq .Invoker "agent-installer") }}
```

Comment:
[Previous Discussion](https://github.com/openshift/installer/pull/7484#pullrequestreview-1631108866).

Reply:
This is only meant as a reference, it is not supposed to block this patch.